defrag/modules/docs.py

```diff
@@ -0,0 +1,159 @@
+from defrag import app
+from defrag.modules.helpers.services_manager import ServiceTemplate, ServicesManager
+from defrag.modules.helpers.requests import Req
+from defrag.modules.helpers import Query, QueryResponse
+from lunr import lunr
+from lunr.index import Index
+from bs4 import BeautifulSoup
+from pydantic import BaseModel
+from concurrent.futures import ProcessPoolExecutor
+import asyncio
+import json
+import sys
+from typing import Any, Dict, List, Tuple
+
+__MOD_NAME__ = "docs"
```

Comment:
```suggestion
__MOD_NAME__ = "documentation"
```

On /docs, FastAPI serves autogenerated documentation